app.vue

```diff
@@ -275,4 +275,32 @@ const downloadCalendar = () => {
 const formatDate = (date: string) => {
   return new Date(date).toLocaleString("de");
 };
+
+// add selected events to URL
+watch(selectedEvents, () => {
+  const selectedEventIds = [...selectedEvents.values()].map(
+    (event) => event.id
+  );
+  const url = new URL(window.location.href);
+  if (selectedEventIds.length === 0) {
+    url.searchParams.delete("events");
+  } else {
+    url.searchParams.set("events", selectedEventIds.join(","));
+  }
+  window.history.replaceState({}, "", url.toString());
```

Comment:
why don't you use `vue-router` instead of manipulating the history stack on your own?

Reply:
Fixed